web/src/components/TtsPlayer.tsx

```diff
@@ -0,0 +1,177 @@
+import React, { ReactElement } from 'react'
+import { useTranslation } from 'react-i18next'
+import styled, { useTheme } from 'styled-components'
+
+import { CloseIcon, PauseIcon, PlaybackIcon, PlayIcon } from '../assets'
+import dimensions from '../constants/dimensions'
+import TtsHelpModal from './TtsHelpModal'
+import Button from './base/Button'
+import Icon from './base/Icon'
+
+const StyledTtsPlayer = styled.dialog<{ $isPlaying: boolean }>`
+  background-color: ${props => props.theme.colors.ttsPlayerBackground};
+  border-radius: 28px;
+  width: 388px;
+  display: flex;
+  flex-direction: ${props => (props.$isPlaying ? 'column' : 'row')};
+  justify-content: center;
+  align-items: center;
+  padding: 6px;
+  position: sticky;
+  bottom: 5px;
+  min-height: 93px;
+  gap: ${props => (props.$isPlaying ? '5px;' : '10px')};
+  border-color: transparent;
+
+  @media ${dimensions.smallViewport} {
+    width: 90%;
+  }
+`
+
+const verticalMargin = 11
+
+const StyledPanel = styled.div<{ $isPlaying?: boolean }>`
+  display: flex;
+  align-items: center;
+  gap: 20px;
+  margin: ${props => (props.$isPlaying ? verticalMargin : 0)}px 0;
+`
+
+const buttonBaseStyles = `
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  transition:
+    box-shadow 0.2s ease,
+    transform 0.1s ease;
+
+  &:active {
+    box-shadow: none;
+    transform: translateY(2px);
+  }
+`
+
+const StyledPlayIcon = styled(Button)`
+  ${buttonBaseStyles}
```

Comment:
🔧 You could make a styled base button that you then use instead of importing the styles, so something like this:

```javascript
const BaseButton = styled(Button)`
  display: flex;
  justify-content: center;
  align-items: center;
  transition:
    box-shadow 0.2s ease,
    transform 0.1s ease;
  &:active {
    box-shadow: none;
    transform: translateY(2px);
  }
`

const StyledPlayIcon = styled(BaseButton)`
...
```